tests/client.py

```diff
@@ -749,6 +751,13 @@ def create(self, stream, company_ids=[], subscriptions=[], times=1):
             return self.create_companies()
         elif stream == 'contact_lists':
             return self.create_contact_lists()
+        elif stream == 'static_contact_lists':
+            staticlist = self.create_contact_lists(False)
+            if stream not in self.record_create_times.keys():
+                self.record_create_times['contacts']=[]
+            records =  self.create_contacts()
+            self.add_contact_to_contact_list()
```

Comment:
We should get the list_id out of the `staticlist` above and send it along here for the method to have.  I think the same is true for the contact_email in the created contact.

Reply:
Done